README.md

```diff
@@ -121,7 +122,27 @@ We're almost done! We just have to tell Netlify how to build our Next.js app, wh
   publish   = "out_publish"
 ```
 
-Note: `out_functions` and `out_publish` are hard-coded into next-on-netlify. These are not configurable at the moment.
+Note: `out_functions` and `out_publish` are hard-coded into next-on-netlify.
+These are not configurable at the moment.
+
+#### (Optional) Configure private git submodules
+If your project contains private submodules, in order to deploy it, you will
+need to:
+
+1. [Generate a Deploy
+   Key](https://docs.netlify.com/configure-builds/repo-permissions-linking/#deploy-keys)
+   in Netlify and [add it to the relevant
+   submodules](https://docs.github.com/en/developers/overview/managing-deploy-keys#setup-2)
+   so that they can be cloned during the deploy process.
+
+2. Ensure the submodule remotes are set to SSH format (i.e.
+   `git@github.com:owner/project.git`, not `https://...`). Inside the submodule
+   directory, the git remote can be updated with: 
+
+   ```bash
+   # git remote set-url [remote] [url]
+   git remote set-url origin git@github.com:owner/project.git
+   ```
```

Comment:
per your comment on the issue about this being "out-of-the-box" on vercel.. is this supported on vercel? only a few months ago this was commented: https://github.com/vercel/vercel/discussions/4566#discussioncomment-125253

Reply:
also: https://vercel.com/docs/build-step#git-submodules

<img width="728" alt="Screen Shot 2021-01-21 at 4 35 18 AM" src="https://user-images.githubusercontent.com/57350178/105331990-1af49c00-5ba2-11eb-8ab2-f0a36363d449.png">

Reply:
Very interesting - all I know is that this project deployed to Vercel w/o issue.

Wonder if they didn't adjust the default permissions for the GH access key so that it would have access to private repos by default, can't think otherwise how it didn't fail.